kapitan/cli.py

```diff
@@ -620,6 +652,11 @@ def secret_update_validate(args, ref_controller):
                         secret_obj.update_key(awskey)
                         ref_controller[token_path] = secret_obj
 
+            elif token_path.startswith("?{vaultkv:"):
+                if not vaultkv:
+                    logger.debug("secret_update_validate: target: %s has no inventory vaultkv parameters 'auth', skipping %s", target_name, token_path)
+                continue
```

Comment:
Is this `continue` indented correctly? Should it be under the `if not vaultkv:` ? 
I'm not sure if that's the case or the github UI is confusing me.

Reply:
That wasn't indented correctly, thanks for pointing that out.